src/appshell/qml/Preferences/internal/UiColorsSection.qml

```diff
@@ -32,8 +32,22 @@ BaseSection {
     title: qsTrc("appshell", "UI colors")
     navigation.direction: NavigationPanel.Both
 
+    property alias scoreInversionChecked: scoreInversionEnable.checked
+
+    signal scoreInversionRequested(var newValue)
     signal colorChangeRequested(var newColor, var propertyType)
 
+    CheckBox {
+        id: scoreInversionEnable
+
+        text: qsTrc("appshell", "Invert score colors")
+
+        onClicked: {
+            checked = !checked
```

Comment:
after this line, your alias doesn't work anymore because you rewrite to a fixed value(just bool, no binding)
property alias scoreInversionChecked: scoreInversionEnable.checked

qml should request about value changed and no more. The model will change the local value and emit a signal about data changed, the view will update the state by alias. But the model can also cancel data changing, then the data of model and of view are different because the view already changed value(checked = !checked).

You can see how I fixed it for highContrastEnabled

Reply:
Ah I see what you mean

I have removed the `checked = !checked` line, and made the alias a bool property. The `onClicked` function now looks like this:
```c++
onClicked: {
            root.scoreInversionRequested(!checked)
        }
```
The system is now almost identical to the High Contrast checkbox.